analyzers/src/SonarAnalyzer.Vsix/SonarAnalyzer.Vsix.csproj

```diff
@@ -58,6 +58,7 @@
     <None Include="app.config" />
   </ItemGroup>
   <ItemGroup>
+    <None Include="packages.lock.json" />
```

Comment:
I think this should not be part of the csproj.

Reply:
I'd also rather delete the whole file 😆 It was autogenerated by VS, because it's old framework-based noisy Csproj.